src/main/java/entity/Message.java

```diff
@@ -41,49 +35,35 @@
 
     public Message() {}
 
-    public Message(long messageId, long chatId, long senderId, String contentType) {
+    public Message(long messageId, long chatId, long senderId) {
         this.messageId = messageId;
         this.chatId = chatId;
         this.senderId = senderId;
-        this.contentType = contentType;
     }
 
-    public Message(long chatId, long senderId, String contentType) {
+    public Message(long chatId, long senderId) {
         this.chatId = chatId;
         this.senderId = senderId;
-        this.contentType = contentType;
     }
 
-    public Message(long messageId, long chatId, long senderId, String contentType, String textContent) {
+    public Message(long messageId, long chatId, long senderId, String textContent) {
         this.messageId = messageId;
         this.chatId = chatId;
         this.senderId = senderId;
-        this.contentType = contentType;
         this.textContent = textContent;
     }
 
-    public Message(long chatId, long senderId, String contentType, String textContent) {
+    public Message(long chatId, long senderId, String textContent) {
         this.chatId = chatId;
         this.senderId = senderId;
-        this.contentType = contentType;
         this.textContent = textContent;
     }
 
-    public Message(long messageId, long chatId, long senderId, String contentType, long contentId, String linkToBlob) {
+    public Message(long messageId, long chatId, long senderId, Long attachmentId) {
```

Comment:
I think you can use `long` instead of `Long` here since it won't be null.

Reply:
I changed it to `Long` since in `createMessage`, if an `Attachment` is not sent, `attachmentId` of the `newMessage` variable is set to a default value of `0`, which violates the `Foreign Key` constraint for `AttachmentID` in `Message`.